code/game/mecha/mecha_construction_paths.dm

```diff
@@ -1606,7 +1606,7 @@
 				user.visible_message("[user] unfastens the scanner module.", "<span class='notice'>You unfasten the scanner module.</span>")
 		if(14)
 			if(diff==FORWARD)
-				user.visible_message("[user] secures [I].", "<span class='notice'>You secure [I].</span>")
+				user.visible_message("[user] secures the capacitor.", "<span class='notice'>You secure the capacitor.</span>")

```

Comment:
What is this fixing? This should be fixed so `I` is still used

Reply:
[I] is the screwdriver, the other mechs use this format

Reply:
Yes I know, I told others in the past to use this format. Why are you changing away from it?

Reply:
A capacitor was already installed to the mech in the previous step. This is the proceeding step where you secure it using a screwdriver. This is correct, unless literally the rest of this entire file is wrong.

Reply:
Oh godamit I was misreading where this was